src/skeleplex/data/tubes.py

```diff
@@ -0,0 +1,62 @@
+"""Drawing tubes in a volume as example data."""
+
+import numpy as np
+from skimage.draw import disk
+
+
+def draw_tube(image, center=(50, 50), diameter=20, z_span=(25, 75)):
+    """
+    Add a tube aligned along the z-axis into an existing 3D volume.
+
+    Parameters
+    ----------
+    image : ndarray
+        3D volume (z, y, x) that will be modified in place.
+    center : tuple of int
+        (row, col) center of the tube in each xy slice.
+    diameter : int
+        Diameter of the circular cross-section.
+    z_span : tuple of int
+        (start, end) range of z-slices to fill.
+
+    Returns
+    -------
+    image : ndarray
+        The input image with the new tube added.
+    """
+    radius = diameter // 2
+    rr, cc = disk(center, radius, shape=image.shape[1:])
+
+    for z in range(z_span[0], z_span[1]):
+        image[z, rr, cc] = 1
+
+    return image
+
+
+def draw_tubes_image_example():
```

Comment:
Please use type annotations for the return type as well.

```suggestion
def draw_tubes_image_example() -> np.ndarray:
```